Lib/gftools/encodings/GF Glyph Sets/GF-latin-core_unique-glyphs.nam

```diff
@@ -1,6 +1,8 @@
-         NULL
+
+#$ include GF-latin-ascii_unique-glyphs.nam
+         .notdef
+         .null
 0x000D   CR
-0x0020   SPACE
```

Comment:
0020 `space` is required
00A0 `nbspace` was the only one to be removed

Reply:
I think it got moved to the top.

Reply:
Didn't find it with `cmnd+f`, not by `SPACE` or `0020`